docs/sql-ref-syntax-dml-insert-overwrite-directory.md

```diff
@@ -39,13 +39,13 @@ INSERT OVERWRITE [ LOCAL ] DIRECTORY [ directory_path ]
 <dl>
   <dt><code><em>file_format</em></code></dt>
   <dd>
-  Specifies the file format to use for the insert. Valid options are <code>TEXT</code>, <code>CSV</code>, <code>JSON</code>, <code>JDBC</code>, <code>PARQUET</code>, <code>ORC</code>, <code>HIVE</code>, <code>DELTA</code>, <code>LIBSVM</code>, or a fully qualified class name of a custom implementation of <code>org.apache.spark.sql.sources.DataSourceRegister</code>.
+  Specifies the file format to use for the insert. Valid options are <code>TEXT</code>, <code>CSV</code>, <code>JSON</code>, <code>JDBC</code>, <code>PARQUET</code>, <code>ORC</code>, <code>HIVE</code>, <code>DELTA</code>, <code>LIBSVM</code>, or a fully qualified class name of a custom implementation of <code>org.apache.spark.sql.execution.datasources.FileFormat</code>.
```

Comment:
Isn't AVRO also valid? Btw, does DELTA implement FileFormat? If it means DeltaDataSource, looks like it only implements DataSourceRegister.

Reply:
Avro implements `FileFormat`. I don't know if DELTA is supported in `INSERT OVERWRITE [ LOCAL ] DIRECTORY`, let me remove it as it's an external source.